css/styles.css

```diff
@@ -143,8 +145,102 @@ aside p {
   padding: 0 40px 0 20px;
 }
 
+.imageBox {
+  padding: 0px 0px 0px 0px;
+}
+
+img {
+  margin-top: 60px;
+}
+
 footer {
   clear: both;
   text-align: center;
   padding: 10px;
 }
+
+#posts {
+  width: 700;
+  display: inline-block;
+  margin-right: 20px;
+}
+
+.post {
+  border-bottom: 1px solid #f2e2cd;
+  padding-bottom: 40px;
+  margin-bottom: 40px;
+}
+
+.readmore, .readless {
+  text-align: right;
+  margin-top: 30px;
+}
+.readmore a, .readless a {
+  color: #f2e2cd;
+  font-weight: 600;
+}
+
+.hide {
+  display: none;
+}
+
+@media (max-width: 900px) {
+
+aside {
+  float: none;
+  width: 100%;
+  padding-bottom: 10px;
+}
+
+aside h2 {
+  font-size: 30px;
+  padding: 20px 40px 20px 20px;
+  text-align: center;
+}
+
+.hubbleBox {
+  visibility: hidden;
+}
+
+main {
+  width: 100%;
+  float: none;
+  padding: 20px;
+}
+
+input {
+  float: left;
+  padding-right: 100px;
+}
+
+}
+
+@media (max-width: 480px) {
+
+header li {
```

Comment:
Indent these too, since they're inside this media query. Also nice use of two media queries to handle tablet and phones!